recipes/ibmpairs/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "ibmpairs" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7bbfec1dc04000aa37235b684173d523ce1480c38419219444e8d2924db8a596
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - fs
+    - future
+    - numpy
+    - pandas
+    - pillow >=1.6
+    - pip
+    - python
+    - pytz
+    - requests >=2.4
+    - shapely
+  run:
+    - fs
+    - future
+    - numpy
+    - pandas
+    - pillow >=1.6
+    - python
+    - pytz
+    - requests >=2.4
+    - shapely
+
+test:
+  imports:
+    - ibmpairs
+  requires:
+    - codecov
+    - coverage
+    - flake8
+    - pytest
+    - pytest-cov
+#    - pytest-ordering
+    - responses
```

Comment:
Hi @marcelotrevisani :

Ok, done by the commit below. Actually, FYI: Would `conda-forge` have `pytest-ordering` I could have gone fancy regarding module testing with `pytest` instead of a simple `import ibmpairs`. But it is fine, since it happens on commit to the `master` branch of https://github.com/ibm/ibmpairs .